tests/math_translator_spec.js

```diff
@@ -5,13 +5,15 @@ const {
     translateMath,
     maybeTranslateMath,
     normalizeTranslatedMath,
+    detectClosedInterval,
+    detectCoordinates,
```

Comment:
I wonder if there's value to have tests just for this block:
```
    if (detectClosedInterval(math)) {
        math = translateIntervals(math, template, lang);
    } else if (detectCoordinates(math)) {
        math = translateCoordinates(math, template, lang);
    } else {
        math = translateCoordinatesOrOpenIntervals(math, template, lang);
    }
```

Or at least for `translateCoordinates` and `translateCoordinatesOrOpenIntervals` in isolation?

Reply:
Hmm, I think I am effectively testing that block by calling `maybeTranslateMath()`.
Testing the individual functions separately would probably make the tests a bit clearer.

Reply:
@mpolyak LMK if you want me to refactor the tests. I can see the value, but I worry a bit that I already spent disproportionate amount of time on this. I'd suggest to get this merged and later deployed and iterate bases on advocate feedback if needed.
(I want to do one last PR before deployment, this PR only needs merging now).

Reply:
That's fair.